piwebapi_samples/Angular/AngularPIWebAPISample/README.md

```diff
@@ -66,14 +66,14 @@ In order to run this sample, you must configure PI Web API with the proper secur
 
 In addition, PI Web API must be configured to allow CORS as follows:
 
-| Attribute               | Value                                               | Type    |
-| ----------------------- | --------------------------------------------------- | ------- |
-| CorsExposedHeaders      | Allow,Content-Encoding,Content-Length,Date,Location | String  |
-| CorsHeaders             | \*                                                  | String  |
-| CorsMethods             | \*                                                  | String  |
-| CorsOrigins             | \*                                                  | String  |
-| CorsSupportsCredentials | True                                                | Boolean |
-| DisableWrites           | False                                               | Boolean |
+| Attribute               | Value                                                                | Type    |
+| ----------------------- | -------------------------------------------------------------------- | ------- |
+| CorsExposedHeaders      | Allow,Content-Encoding,Content-Length,Date,Location                  | String  |
+| CorsHeaders             | \*                                                                   | String  |
+| CorsMethods             | \*                                                                   | String  |
+| CorsOrigins             | https://localhost:4000,https://localhost:9876,https://localhost:8000 | String  |
```

Comment:
This is OK, but should we consider hosting all the piwebapi samples on the same port to avoid listing these three random localhost ports? I'm pretty sure each sample could be updated to use a specific port. Also, we're saying these CorsOrigins are required for every sample, but in reality, at most one of these origins is required for each sample.

Reply:
You're correct that only one of the ports is needed for each, but we thought it'd be better to keep the configuration consistent across all readmes for simplicity. If you'd prefer to list a single port for each readme, I'd lean toward listing the relevant default port for that language rather than adding instructions for standardizing the port used.

Reply:
What I'm suggesting is to update the samples themselves so that they all use the same port number, then just list that port number in the CorsOrigins. I'm pretty confident that at least for the Angular and NodeJS, it's possible to specify the port number, so if we were consistent we could avoid this long CorsOrigins list.

Reply:
Yeah, understood. If you want to avoid listing multiple CorsOrigins, then I'd prefer to tailor the readme to the language. Ie. Angular would list only 9876, R and Python would not list anything...

Should I make that change?

Reply:
What I mean is like this:
https://github.com/osisoft/OSI-Samples-PI-System/pull/52
CorsOrigins is still the same for all the samples, but the same port is used by every sample.

Reply:
Yes, I understand your suggestion. It seems like it would be easier to use default ports so that we don't also have to update the readmes to add instructions for changing ports. Is it ok to either:

1. Leave as-is?
2. Specify the valid default port for each language in the CorsOrigins setting?

Or do you feel strongly that we should make them consistent and add instructions for setting the port for each language?

Reply:
If you take a look at my PR, what I'm suggesting is none of these things; we can simply change the ports that are used by the samples to all use the same port, then we pretty much don't have to change the readmes because the samples themselves specify the port to use.

Reply:
I updated to use only 9876. Is this what you had in mind?